frontend/src/components/NewPassword/NewPassword.js

```diff
@@ -10,39 +10,41 @@ class NewPassword extends Component {
       isPasswordVisible: false,
       password: '',
       confirmPassword: '',
-      error: '',
+      error: ''
```

Comment:
Whatever setting is causing VSCode to remove trailing commas, you should turn it off. It's causing many (though not all) of the linter errors currently causing your build to fail.